docs-md/README.md

```diff
@@ -45,6 +45,7 @@
 * Web/PWA
   * [Getting Started](web/index.md)
   * [PWA Elements](web/pwa-elements.md)
+  * [Utilities](web/utilities.md)
```

Comment:
While you access these methods from JS, this is useful in terms of development helpers. still the best spot?